torch_xla/csrc/runtime/xla_coordinator.cc

```diff
@@ -9,6 +9,9 @@ namespace runtime {
 
 XlaCoordinator::XlaCoordinator(int global_rank, int world_size,
                                std::string master_addr, std::string port) {
+  if (world_size <= 1) {
```

Comment:
Can we instead avoid creating the client specifically for GPUs in the PjRtComputationClient constructor? Autocheckpointing will still require a coordinator even with world size 1.

Reply:
Sure.